src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java

```diff
@@ -432,7 +432,10 @@ final Charset charset() {
      *
      * @see #debuglevel
      * @see <a href="https://docs.oracle.com/en/java/javase/17/docs/specs/man/javac.html#option-g">javac -g</a>
+     *
+     * @deprecated Setting this flag to {@code false} is replaced by {@code <debuglevel>none</debuglevel>}.
```

Comment:
Should we also update description for `debuglevel` parameter?

Reply:
Done, thanks for the reminder.